config/gatsby/plugins/sass.js

```diff
@@ -0,0 +1,27 @@
+const path = require("path")
+const postCssUrl = require("postcss-url")
+const sass = require("sass")
+
+module.exports = (root) => [
+  {
+    resolve: "gatsby-plugin-sass",
+    options: {
+      cssLoaderOptions: {
+        localIdentName: "[folder]-[name]--[local]--[hash:base64:5]",
```

Comment:
I'm confused, is this dev only? Because in prod you only need the `hash` part.

Reply:
This is used in both places. GatsbyJS doesn't seem to make any distinction between the two in the CSS Loader configuration, probably due to the fact the full hash produces bigger classes than this setup (https://github.com/webpack-contrib/css-loader/issues/406).

Since GatsbyJS doesn't make the distinction I didn't bother with it, but we distinguish it ourselves. Would you prefer to use `[hash:base64:20]` in production? Or any other hash length?